feature/experimental/qos/egress_strict_priority_scheduler_with_bursty_traffic/README.md

```diff
@@ -0,0 +1,158 @@
+
+## QoS traffic test setup:
+
+*   Topology:
+
+    *   2 input interfaces and 1 output interface with the same port speed. The
+        interface can be a physical interface or LACP bundle interface with the
+        same aggregated speed.
+
+    ```
+      ATE port 1
+          |
+         DUT--------ATE port 3
+          |
+      ATE port 2
+    ```
+
+*   Traffic classes:
+
+    *   We will use 6 traffic classes NC1, AF4, AF3, AF2, AF1 and BE1.
+
+*   Traffic types:
+
+    *   All the traffic tests apply to both IPv4 and IPv6 and also MPLS traffic.
+
+*   Queue types:
+
+    *   NC1/AF4/AF3/AF2/AF1/BE1 will have strict priority queues (be1 - priority 6, af1 - priority 5, ..., nc1 - priority 1)
+
+*   Test results should be independent of the location of interfaces. For
+    example, 2 input interfaces and output interface could be located on
+
+    *   Same ASIC-based forwarding engine
+    *   Different ASIC-based forwarding engine on same line card
+    *   Different ASIC-based forwarding engine on different line cards
+
+*   Test results should be the same for port speeds 100G and 400G.
+
+*   Counters should be also verified for each test case:
+
+    *   /qos/interfaces/interface/output/queues/queue/state/transmit-pkts
+    *   /qos/interfaces/interface/output/queues/queue/state/dropped-pkts
+    *   transmit-pkts should be equal to the number of Rx pkts on Ixia port
+    *   dropped-pkts should be equal to diff between the number of Tx and the
+        number Rx pkts on Ixia ports
+
+*   Latency:
+
+    *   Should be < 100000ns
+
+## Procedure
+
+* DUT Configuration:
+    * Forwarding Classes: Configure six forwarding classes (be1, af1, af2, af3, af4, nc1) based on the classification table provided.
+    * Egress Scheduler: Apply a multi-level strict-priority scheduling policy on the desired egress interface. Assign priorities to each forwarding class as below: 
+        * be1 - priority 6
+        * af1 - priority 5
+        * af2 - priority 4
+        * af3 - priority 3
+        * af4 - priority 2
+        * nc1 - priority 1
+
+* Classification table
+
+    IPv4 TOS      |       IPv6 TC           |         MPLS EXP        |    Forwarding class
+    ------------- | ----------------------- | ----------------------- | ---------------------
+    0             |      0-7                |          0              |         be1
+    1             |      8-15               |          1              |         af1
+    2             |      16-23              |          2              |         af2
+    3             |      24-31              |          3              |         af3
+    4,5           |      32-47              |          4,5            |         af4
+    6,7           |      48-63              |          6,7            |         nc1
+
+* Traffic Generation:
```

Comment:
Generate steady state traffic from ATE Port 1. Take the traffic profile from QOS-1.3

Generate bursty traffic from ATE Port 2 for all traffic classes simultaneously with the same profile for each class:
Frame size: 256
Inter-packet gap: 12
Burst size: 50000
Inter-burst gap: 100ms

Verify that no high priority traffic is dropped (AF3/AF4).